spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/README.adoc

```diff
@@ -0,0 +1,54 @@
+= Spring Cloud GCP Secret Manager Sample Application
+
+This code sample demonstrates how to use the Spring Cloud GCP Secret Manager integration.
+
+To begin using Secret Manager support, first add the `spring-cloud-gcp-starter-secretmanager` dependency to the project:
+
+```
+<dependency>
+	<groupId>org.springframework.cloud</groupId>
+	<artifactId>spring-cloud-gcp-starter-secretmanager</artifactId>
+</dependency>
+```
+
+Adding the starter dependency will automatically cause your GCP Project's secrets to be injected into your application's environment at bootstrap time.
+
+This sample application defines a `@ConfigurationProperties` in order to access and use the secret values.
+
+```
+@ConfigurationProperties
+public class MyAppProperties {
+
+	@Value("${application-secret}")
```

Comment:
`@Value` is usually used outside of the `@ConfigurationProperties`. I would instead demonstrate the use of a prefix like in the `spring-cloud-gcp-config-sample`.

Reply:
Since you no longer use the `@ConfigurationProperties`, this section needs to be amended.

Reply:
Done; instead of duplicating code snippet I just get a general summary of sample contents.